src/pytom_tm/matching.py

```diff
@@ -101,7 +116,8 @@ def __init__(
         - pyTME: https://github.com/KosinskiLab/pyTME
 
         The precalculation of conjugated FTs of the tomo was (AFAIK) introduced
-        by STOPGAP!
+        by STOPGAP! Also, they introduced simultaneous matching with a phase randomized
+        version of the template.
```

Comment:
maybe also add doi link here?
```suggestion
        version of the template. https://doi.org/10.1107/S205979832400295X
```